File: 171-excel-sheet-column-number/excel-sheet-column-number.java
class Solution {
    public int titleToNumber(String c) {
        int ans = 0;
        for (int i = c.length() - 1; i >= 0; i-- ) {
            int power = c.length() - 1 - i;
            ans += ((c.charAt(i) - 'A' + 1) * (int)Math.pow(26, power));
        }
        return ans;
    }
}

File: 171-excel-sheet-column-number/excel-sheet-column-number.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        mul = 1
        val = 0
        for i in range(len(columnTitle) - 1, -1, -1):
            val += (ord(columnTitle[i]) - 65 + 1) * mul
            mul *= 26 
        return val